pkg/codegen/docs/gen.go

```diff
@@ -329,13 +322,12 @@ func (ss nestedTypeUsageInfo) contains(token string, input bool) bool {
 }
 
 type modContext struct {
-	pkg          *schema.Package
-	mod          string
-	resources    []*schema.Resource
-	functions    []*schema.Function
-	children     []*modContext
-	tool         string
-	emitAPILinks bool
```

Comment:
I debated keeping this property around and hence the code that uses it and simply default to passing `false` so that the links wouldn't be emitted. But I felt it was unnecessary to do so. If we ever do go back to generating the doc links I think a fresh start would be good given the state of things at _that_ time with regards to how the links should be generated. Anyway, deleting this property caused me to delete a lot of conditional code and also code that we just won't need anymore.